### ISSUE TITLE
telegraf:latest has no pgrep

### ISSUE THREAD

**Issue by sheb-gregor** [influxdata/influxdata-docker#166]
Hello!
I tried to start with the telegraf in the docker with the  enabled `[[inputs.procstat]]` and got the following error:

```
2017-11-15T16:58:30Z E! Error in plugin [inputs.procstat]: E! Error: procstat getting process, exe: [] pidfile: [] pattern: [.*] user: [] Could not find pgrep binary: exec: "pgrep": executable file not found in $PATH
```
Part of my `docker-compose.yml`:

```yml
version: '2'
services:
  telegraf:
    image: telegraf:latest
    container_name: telegraf
```
If you use ` telegraf:1.4-alpine` then everything this error not present.

**Comment by hasnat** [comment]
I believe this is fixed here influxdata/influxdata-docker#148

**Comment by sheb-gregor** [comment]
Great! I think that issue can be closed  :-) :+1:
